hydroshare/hydroshare
# Fixing scimeta root endpoint

Fixes #2544 

To test: 

1. Check out the branch and restart (no need to rebuild)
2. Visit /hsapi locally to see the Swagger docs
3. Expand hsapi
4. Scroll down to GET /hsapi/resource/{id}/scimeta/
5. Expand that section and insert any public resource ID into ID
6. Click "try it out!"
7. RDF metadata should appear in the response
